setup.sh

```diff
@@ -118,6 +118,8 @@ if [[ -n $JAX_VERSION && ! ($MODE == "stable" || -z $MODE || ($MODE == "nightly"
      exit 1
 fi
 
+uv pip install -r './extra_deps_from_github.txt'
+
```

Comment:
Shouldn't this go after `python3 -m uv pip install --resolution=lowest -r "$tpu_requirements_txt"`?

Reply:
It could. I put it earlier so that the dependencies wouldn't conflict with pinned version dependencies later. Your call.